### PR TITLE
feat(preview/tooltip): pass preview experiences to preview

### PR DESCRIPTION
**Pull request by megansmith-box**
Pass PreviewExperiences From ContentPreview Component to Preview Options that will be used by Content Preview

### CODE REVIEW

File: src/components/tooltip/Tooltip.scss [hunk 1]
```diff
@@ -64,6 +64,11 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
         }
     }
 
+    &.is-wide {
```
Comment (jstoffan) [inline src/components/tooltip/Tooltip.scss:67]: Is this change related to the preview changes or should it be a separate PR? We will need to very cautious about (or even avoid entirely) using BUIE components in Preview SDK, since it can lead to styling issues.

File: src/components/tooltip/Tooltip.scss [hunk 2]
```diff
@@ -88,6 +93,23 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
         }
     }
 
+    .tooltip-callout {
+        display: flex;
+        flex-flow: column;
+        font-weight: normal;
+        font-size: 13px;
+        line-height: 1.54;
```
Comment (jstoffan) [inline src/components/tooltip/Tooltip.scss:101]: Can we use a more standardized value to avoid this magic number?

File: src/components/tooltip/Tooltip.scss [hunk 3]
```diff
@@ -88,6 +93,23 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
...
+        letter-spacing: .3px;
```
Comment (jstoffan) [inline src/components/tooltip/Tooltip.scss:102]: Is this needed or can we use the default kerning?

File: src/components/tooltip/Tooltip.scss [hunk 4]
```diff
@@ -88,6 +93,23 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
...
+        cursor: default;
```
Comment (jstoffan) [inline src/components/tooltip/Tooltip.scss:103]: Is this needed?

File: src/elements/content-preview/ContentPreview.js [hunk 5]
```diff
@@ -1274,6 +1283,7 @@ class ContentPreview extends React.PureComponent<Props, State> {
                             onDownload={this.download}
                             contentOpenWithProps={contentOpenWithProps}
                             canAnnotate={this.canAnnotate()}
+                            previewExperiences={previewExperiences}
```
Comment (jstoffan) [inline src/elements/content-preview/ContentPreview.js:1286]: This version of the preview header is only shown to platform customers, so this may not be needed.

Comment (megansmith-box) [reply src/elements/content-preview/ContentPreview.js:1286]: yes, I will remove, thanks!

File: src/components/tooltip/Tooltip.scss (hunk 3 above)
Comment (megansmith-box) [reply src/components/tooltip/Tooltip.scss]: yes, it's not needed, thank you

File: src/components/tooltip/Tooltip.scss (hunk 4 above)
Comment (megansmith-box) [reply src/components/tooltip/Tooltip.scss]: Yes, the cursor changes to select text cursor versus pointer cursor without this.

File: src/components/tooltip/Tooltip.scss (hunk 2 above)
Comment (megansmith-box) [reply src/components/tooltip/Tooltip.scss]: I copied the styles from left-nav-callout css style. I'm thinking at some point we can replace left-nav-callout with this more generic class tooltip-callout. If I remove it it looks more condensed than the given style. And If i increase it, it looks more spread apart. I think I will keep it as close to design as possible and consistent with the left-nav-callout unless you think it's super important and should be changed

File: src/elements/content-preview/ContentPreview.js [hunk 6]
```diff
@@ -784,6 +789,7 @@ class ContentPreview extends React.PureComponent<Props, State> {
             fileOptions: fileOpts,
             header: 'none',
             headerElement: `#${this.id} .bcpr-PreviewHeader`,
+            previewExperiences,
```
Comment (jstoffan) [inline src/elements/content-preview/ContentPreview.js:792]: Minor, but since we're passing this value to preview, should we go with just `experiences` here?

Comment (megansmith-box) [reply src/elements/content-preview/ContentPreview.js:792]: yes, I like that idea

**Review by jstoffan — APPROVED** [review]

File: src/components/tooltip/Tooltip.scss (hunk 2 above)
Comment (alexkrolick) [reply src/components/tooltip/Tooltip.scss]: 1.54 * 13px = 20px, which was probably meant to be the height of the element
